pkg/router/functionHandler.go

```diff
@@ -45,8 +45,8 @@ const (
 	// FORWARDED represents the 'Forwarded' request header
 	FORWARDED = "Forwarded"
 
-	// X_FORWARDED_HOST represents the 'X_FORWARDED_HOST' request header
-	X_FORWARDED_HOST = "X-Forwarded-Host"
+	// XForwardedHost represents the 'XForwardedHost' request header
```

Comment:
This should be `X_FORWARDED_HOST` only , the header name. Probably instance of wrong string replacement.